packages/@okta/vuepress-site/docs/reference/api/threat-insight/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: ThreatInsight
+category: management
+---
+
+# ThreatInsight Configuration API
+
+[Okta ThreatInsight](https://help.okta.com/en/prod/Content/Topics/Security/security-threat-insight.htm) maintains a constantly evolving list of IPs that exhibit suspicious behaviors suggestive of malicious activity. Authentication requests associated with an IP in this list can be logged in [System Log](https://help.okta.com/en/prod/Content/Topics/Reports/Reports_SysLog.htm?cshid=ext_Reports_SysLog) and blocked. The Okta ThreatInsight Configuration API provides operations to manage your ThreatInsight configuration.
+
+
+## ThreatInsight Configuration object
+
+| Field Name     | Description                                                         	| Data Type                                     | Required      | Max Length    |
+| :------------- | :------------------------------------------------------------------	| :-------------------------------------------- | :------------ | :------------ |
+| action         | Specifies how Okta would respond to authentication requests from suspicious IPs. Value is one of `none`, `audit`, `block`. With `none`, ThreatInsight will be disabled; with `audit`, Okta will log the suspicious requests in System Log; with `block`, Okta will log the suspicious requests in System Log and block the requests. | String                                        | Yes		| N/A           |
```

Comment:
a couple of edits for clarity:

Specifies how Okta responds to authentication requests from suspicious IPs. Values are `none`, `audit`, or `block`. A value of `none` indicates that ThreatInsight is disabled. A value of `audit` indicates that Okta logs suspicious requests in the System Log. A value of `block` indicates that Okta logs suspicious requests in the System Log and blocks the requests.